### PR TITLE
#1 Initial commit

### PR DESCRIPTION
**Pull request by gscdev**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -34,7 +37,9 @@ captures/
 
 # Intellij
 *.iml
-.idea/workspace.xml
+.idea
```
Comment (jogrimst) [inline .gitignore]: Is `.idea` the same as `*.idea`?

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  - oraclejdk8
+
+android:
+  components:
+    - tools
+    - build-tools-23.0.3
+    - android-23
+    - extra-android-m2repository
+
+#notifications:
```
Comment (jogrimst) [inline .travis.yml]: It seems like e-mails are sent automatically to the GitHub organization owner (info@literacyapp.org) on each build when `notifications` is missing, as as default notification mechanism. So update to the following:

```
notifications:
  email: false
```

File: app/build.gradle [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+        minSdkVersion 21
+        targetSdkVersion 23
+        versionCode 1000000
+        versionName "1.0.0"
+        testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"
+    }
+    buildTypes {
+        release {
+            minifyEnabled false
+            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
```
Comment (jogrimst) [inline app/build.gradle]: I think we can delete this line, as well as the proguard-rules.pro file. Or is proguard necessary for anything?

Comment (gscdev) [inline app/build.gradle]: No, proguard is not necessary. I have deleted the line.

File: .gitignore (hunk 1 above)
Comment (gscdev) [inline .gitignore]: They are different. With `*.idea` you can ignore files with a name and with extension .idea _filename.idea_, but not with `.idea`

**Pull request merged**